src/api/v1/index.js

```diff
@@ -6,6 +6,7 @@ const router = express.Router();
 // private API, requires authentication
 router.use('/attendance', authenticated, require('./attendance').router);
 router.use('/leaderboard', authenticated, require('./leaderboard').router);
+router.use('/static', authenticated, require('./static').router);
```

Comment:
Make this `/admin` since we'll be moving some of the `/user` routes into the admin controller (e.g. point bonuses and milestones).